community/php5-xdebug/APKBUILD

```diff
@@ -0,0 +1,41 @@
+# Contributor: Andy Postnikov <apostnikov@gmail.com>
+# Contributor: Valery Kartel <valery.kartel@gmail.com>
+# Contributor: Fabio Ribeiro <fabiorphp@gmail.com>
+# Maintainer: Andy Postnikov <apostnikov@gmail.com>
+pkgname=php5-xdebug
+pkgver=2.5.1
+pkgrel=0
+_pkgver=${pkgver/_rc/RC}
+_phpver=${pkgname%%-*}
+_pkgreal=${pkgname#*-}
+pkgdesc="$_phpver extension: provides functions for function traces and profiling"
+url="http://pecl.php.net/package/$_pkgname"
+arch="all"
+license="PHP"
+pecldepends="$_phpver-dev autoconf"
+makedepends="$pecldepends"
+source="http://pecl.php.net/get/$_pkgreal-$_pkgver.tgz"
+builddir="$srcdir/$_pkgreal-$_pkgver"
+
+build() {
+	cd "$builddir"
+	phpize || return 1
+	./configure --prefix=/usr || return 1
+	make
+}
+
+check() {
+	make -C "$builddir" NO_INTERACTION=1 test
+}
+
+package() {
+	local confdir=$(php-config|grep -o "scan-dir=[[:graph:]]*"|cut -d= -f2)
```

Comment:
agree...
for php7 (7.1) I want to make a helper that do all this grep/cut and just shows confdir or apiver values. And put this helper to php7-pecl subpackage.